SKILL.tsv

```diff
@@ -1089,35 +1089,35 @@ SKILL_20150317_001088	Fire Ball
 SKILL_20150317_001089	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Summon Fire sphere. You can Blow the sphere using sword attack.
 SKILL_20150317_001090	Attack #{SkillAtkAdd}#{nl}Attack #{SkillSR}# times{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001091	Fire Wall
-SKILL_20150317_001092	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Create flame barrier to damage adjacent enemies.
+SKILL_20150317_001092	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Creates a flame barrier inflicting damage to a nearby enemy.
 SKILL_20150317_001093	Attack #{SkillAtkAdd}#{nl}#{CaptionRatio}# Fire Wall #{nl}Attack #{CaptionRatio2}# times{nl}Duration 15 seconds
 SKILL_20150317_001094	Enchant Fire
-SKILL_20150317_001095	Grant Fire Property attack force to target. Additional damage is applied for basic attack.
+SKILL_20150317_001095	Grants the Fire attack attribute to the target. Additional damage applies to your basic attack.
 SKILL_20150317_001096	Fire Property attack + #{CaptionRatio}#{nl} Additional Damage + #{CaptionRatio2}#{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001097	Flare
-SKILL_20150317_001098	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Explode enemy in abnornal state except petrified and frozen state.
+SKILL_20150317_001098	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Inflict damage by exploding the enemy affected by a status effect. Excluding both petrified and frozen status effects.
 SKILL_20150317_001099	Attack #{SkillAtkAdd}#
 SKILL_20150317_001100	Fire Pillar
 SKILL_20150317_001101	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl}Summon fire pillar to damage enemies.
 SKILL_20150317_001102	Attack #{SkillAtkAdd}#{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001103	Hell Breath
-SKILL_20150317_001104	{#993399}{ol}[Magic] - [Fire]{/}{/}{nlContinuously fume flame forward.
+SKILL_20150317_001104	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Continuously burn enemies with a cone of flame.
 SKILL_20150317_001105	Attack #{SkillAtkAdd}#{nl}Consume #{CaptionRatio}#% SP per 0.5 seconds
 SKILL_20150317_001106	Ice Bolt
-SKILL_20150317_001107	{#993399}{ol}[Magic] - [Ice] - [Stab]{/}{/}{nl}Blow Ice force with chance of freezing enemy.
+SKILL_20150317_001107	{#993399}{ol}[Magic] - [Ice] - [Stab]{/}{/}{nl} Inflict damage to a target by shooting a bolt of ice. There is a chance to freeze the target.
 SKILL_20150317_001108	Attack #{SkillAtkAdd}#{nl}{#339999}{ol}[Frozen]{/}{/} Duration 3 seconds
 SKILL_20150317_001109	Ice Pike
-SKILL_20150317_001110	{#993399}{ol}[Magic] - [Ice] - [Stab] {/} {/}} {nl}Create ice waves with chance of freezing enemy.
+SKILL_20150317_001110	{#993399}{ol}[Magic] - [Ice] - [Stab] {/} {/}} {nl} Inflict damage to a target by causing a wave of ice. There is a chance to freeze the target.
```

Comment:
Pretty much as above. "Launch a wave of ice that inflicts damage and has a chance to freeze the target"?